lib/hexpm_web/templates/docs/faq.html.md

```diff
@@ -9,6 +9,12 @@ username we recommend [creating a new account](/signup).
 
 You can transfer ownership of your packages using the [`hex.owner`](https://hexdocs.pm/hex/Mix.Tasks.Hex.Owner.html) task.
 
+#### Password security
```

Comment:
This is awesome 👍 

Can we link to this section from the flash message?

Reply:
yeah, let me figure out how to do that.